plugins/modules/docker_plugin.py

```diff
@@ -17,6 +17,7 @@
 description:
   - This module allows to install, delete, enable and disable Docker plugins.
   - Performs largely the same function as the C(docker plugin) CLI subcommand.
+  - The C(--grant-all-permissions) CLI flag is true by default in this module.
```

Comment:
To move this to the `Notes` section:
```suggestion
notes:
  - The C(--grant-all-permissions) CLI flag is true by default in this module.
```

Reply:
Thanks, Felix! It's not _quite_ as obvious this way, but I do think it's more appropriate for this to be in the notes than the synopsis 👍